projects/solar/src/solar_config.c

```diff
@@ -132,8 +143,11 @@ const SpiSettings *config_get_spi_settings(void) {
   return &s_spi_settings;
 }
 
-const CanSettings *config_get_can_settings(void) {
-  return &s_can_settings;
+const CanSettings *config_get_can_settings(SolarMpptCount mppt_count) {
+  if (mppt_count == SOLAR_BOARD_6_MPPTS)
+    return &s_can_settings_6_mppts;
+  else
+    return &s_can_settings_5_mppts;
```

Comment:
The style guide does say to use curly brackets even with single line statements, really the only exception is for really short one line if statements without an else. Please use curly brackets